src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,15 @@
-export const UserInfo = () => {};
+export interface User {
+  id: number;
+  name: string;
+  email: string;
+}
+
+interface UserInfoProps {
+  user: User;
+}
+
+export const UserInfo: React.FC<UserInfoProps> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
+  </a>
```

Comment:
you can remove optional chaining because here user cannot be undefined (you check it before rendering UserInfo)
```suggestion
  <a className="UserInfo" href={`mailto:${user.email}`}>
    {user.name}
  </a>
```